linode/manager
# M3-5083: Dismissible Images pricing banner

## Description
Adds a dismissible banner that explains Images pricing to the Images landing and and Images creation pages. The displaying of the banner is conditional upon the feature flag and whether the user has images or not.

## How to test
The feature flag is on in the testing environment, so please ensure that the banner does not display on Images pages when you have no images, and that the banner does display on Images pages when you do have images.

The banner should not return on its own after being dismissed (if you dismiss it and want to bring it back for further testing, you can hit the `profile/settings?preferenceEditor=true` page and remove the corresponding entry from `dismissed_notifications`).
